src/settings/window_size.rs

```diff
@@ -1,5 +1,6 @@
 use std::path::PathBuf;
 
+use glutin::dpi::PhysicalSize;
```

Comment:
This sould be `winit::dpi::PyshicalSize`, since already changed all the other references. `winit::dpi::PhysicalPosition` is also used just below this.

Reply:
Fixed :)